flight/drone/drone_controller.py

```diff
@@ -16,7 +16,10 @@
 from flight import constants as c
 from drone import Drone
 import exceptions
-from flight.tasks import Hover, Takeoff, LinearMovement, Land, Exit, TakeoffSim
+
+import config
+from flight import constants as c
+from flight.tasks import Hover, Takeoff, LinearMovement, Land, Exit, TakeoffSim, TaskFactory
```

Comment:
Most of these should not be required now, correct? If we are only passing tasks into the DroneController, then it shouldn't need to know how to make any of these except for Hover